sql/src/test/java/io/crate/integrationtests/SubSelectIntegrationTest.java

```diff
@@ -588,4 +588,63 @@ public void testSubqueryExpressionWithInPredicateEvaluatesToNull() throws Except
         assertThat(response.rowCount(), is(1L));
         assertNull(response.rows()[0][0]);
     }
+
+    @Test
+    public void testNestedSubqueryWithAggregatesInMultipleStages() throws Exception {
```

Comment:
How about `testNestedSemiJoinWithAggregatesInMultipleStages`?

Reply:
@mxm semi-joins not in master yet, this is just 2level multiphase-execution, or am I missing something?

Reply:
@mxm @matriv yes indeed. That's why I don't want to name it to semi-joins.